_includes/header.html

```diff
@@ -10,37 +10,64 @@
+                    <img src="{{site.baseurl}}/assets/nav_logo.svg" style="height:44px;">
                 </a>
             </div>
             <!-- Collect the nav links, forms, and other content for toggling -->
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav navbar-right">
                     {% for nav in site.data.nav.head %}
                       {% if nav.url %}
-                           <li>
-                             <a href="{{site.baseurl}}{{nav.url}}" class="tab" id="tab" target="_blank">{{nav.title | upcase}}</a>
+                           <li id="maintabs">
+                             <a href="{{site.baseurl}}{{nav.url}}" class="tab" id="tab" target="_blank">{{nav.title }}</a>
                            </li>
+                      {% elsif nav.subitem %}
+                          <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ nav.title }} <img id="carrot" src="assets/carrot.svg" alt="carrot"></a>
+                              <ul class="dropdown-menu">
+                                  {% for submenu in nav.subitem %}
+                                    {% if submenu.url %}
+                                      <li><a href="{{ submenu.url }}" target="_blank">{{ submenu.title }}</a></li>
+                                    {% else %} 
+                                      {% for targetpage in site.pages %}
+                                         {% if submenu.title contains targetpage.title%}
+                                            {% if page.title == targetpage.title %}
+                                              <li>
+                                                <a href >{{targetpage.title}}</a>
+                                              </li>
+                                            {% else %}
+                                              <li>
+                                                <a href="{{site.baseurl}}{{targetpage.url}}">{{targetpage.title}}</a>
+                                              </li>
+                                            {% endif %}
+                                        {% endif %}
+                                      {% endfor %}
+                                    {% endif %}
+                                  {% endfor %}
+                              </ul>
+                          </li>
                       {% else %}
                         {% for targetpage in site.pages %}
-                           {% if nav contains targetpage.title%}
+                           {% if nav.title contains targetpage.title%}
                               {% if page.title == targetpage.title %}
-                               <li>
-                                   <a href  class="tab navbar-active" id="tab">{{targetpage.title| upcase}}</a>
+                               <li id="maintabs">
```

Comment:
Same thing for `#maintabs` and `#tab`: cf [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)

> The id global attribute defines a unique identifier (ID) which must be unique in the whole document. Its purpose is to identify the element when linking (using a fragment identifier), scripting, or styling (with CSS).

It **should** appear only once.